flake.nix

```diff
@@ -87,18 +87,37 @@
     flake-compat.flake = false;
   };
 
-  outputs = { self, nixpkgs, flake-utils, emacs-overlay, ... }@inputs:
-    let inherit (flake-utils.lib) eachDefaultSystem eachSystem;
+  outputs = { self, nixpkgs, flake-utils, ... }@inputs:
+    let inherit (flake-utils.lib) eachDefaultSystem mkApp;
     in eachDefaultSystem (system:
       let pkgs = import nixpkgs { inherit system; };
       in {
-        devShell = pkgs.mkShell {
+        apps = {
+          default = self.outputs.apps.${system}.doom-emacs-example;
+          doom-emacs-example = mkApp {
+            drv = self.outputs.packages.${system}.doom-emacs-example;
+            exePath = "/bin/emacs";
+          };
+        };
+
+        devShells.default = pkgs.mkShell {
           buildInputs =
             [ (pkgs.python3.withPackages (ps: with ps; [ PyGithub ])) ];
         };
-        package = { dependencyOverrides ? { }, ... }@args:
-          pkgs.callPackage self
-          (args // { dependencyOverrides = (inputs // dependencyOverrides); });
+
+        package = { ... }@args:
+          pkgs.lib.warn ''
+            Deprecated, will be removed after NixOS 23.05 release.
```

Comment:
```suggestion
            nix-doom-emacs no longer supports the deprecated `package` flake output. It will be removed after the release of NixOS 23.05.
```

Reply:
Done.